Starkcore/utils/Request.cs

```diff
@@ -0,0 +1,125 @@
+            {
+                return System.Text.Encoding.UTF8.GetString(ByteContent);
+            }
+        }
+
+        internal JObject Json()
+        {
+            return Utils.Json.Decode(Content);
+        }
+    }
+
+    internal static class Request
+    {
+        private static HttpClient makeClient()
+        {
+            HttpClient client = new HttpClient();
+            return client;
+        }
+
+        private static readonly HttpClient Client = makeClient();
+        internal static readonly HttpMethod Get = new HttpMethod("GET");
+        internal static readonly HttpMethod Put = new HttpMethod("PUT");
+        internal static readonly HttpMethod Post = new HttpMethod("POST");
+        internal static readonly HttpMethod Patch = new HttpMethod("PATCH");
+        internal static readonly HttpMethod Delete = new HttpMethod("DELETE");
+
+        internal static Response Fetch(User user, HttpMethod method, string path, string host, string apiVersion, string sdkVersion, Dictionary<string, object> payload = null, Dictionary<string, object> query = null)
+        {
+            user = Checks.CheckUser(user);
+
+            string url = "";
+            if (user.Environment == "production")
+            {
+                url = "https://api.stark" + host + ".com/";
+            }
+            if (user.Environment == "sandbox")
+            {
+                url = "https://sandbox.api.stark" + host + ".com/";
+            }
+            url += apiVersion + "/" + path;
+
+            if (query != null)
+            {
+                url += Url.Encode(query);
+            }
+
+            string accessTime = DateTime.UtcNow.Subtract(new DateTime(1970, 1, 1)).TotalSeconds.ToString(new CultureInfo("en-US"));
+            string body = "";
+            if (payload != null)
+            {
+                body = Json.Encode(payload);
+            }
+            string message = user.AccessId() + ":" + accessTime + ":" + body;
+            string signature = Ecdsa.sign(message, user.PrivateKey()).toBase64();
+
+            HttpRequestMessage httpRequestMessage = new HttpRequestMessage
+            {
+                Method = method,
+                RequestUri = new Uri(url)
+            };
+            if (body.Length > 0)
+            {
+                httpRequestMessage.Content = new StringContent(body);
+            }
+
+            httpRequestMessage.Headers.TryAddWithoutValidation("Access-Id", user.AccessId());
+            httpRequestMessage.Headers.TryAddWithoutValidation("Access-Time", accessTime);
+            httpRequestMessage.Headers.TryAddWithoutValidation("Access-Signature", signature);
+            httpRequestMessage.Headers.TryAddWithoutValidation("Content-Type", "application/json");
+            httpRequestMessage.Headers.TryAddWithoutValidation("Accept-Language", Settings.Language);
+            httpRequestMessage.Headers.TryAddWithoutValidation("User-Agent", $".NET-{Environment.Version}-SDK-Infra-{sdkVersion}");
```

Comment:
Você adicionou na linha 100 oq comentei sobre o makeClient, mas se for assim o método makeClient() só morre, já que vc adiciona aqui na lista que vai para o client.